Makefile

```diff
@@ -1,9 +1,10 @@
 PROJECT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CI_BUILD_NUMBER ?= $(USER)-snapshot
 
-VERSION ?= 0.39.0
+VERSION ?= 0.0.$(CI_BUILD_NUMBER)
+FLOW_VERSION ?= 0.40.0
 
-PUBLISH_TAG=meetup/node-flow:$(VERSION)
+PUBLISH_TAG=meetup/node-flow:$(FLOW_VERSION)-$(VERSION)
```

Comment:
So I got this versioning advise from EE - they recommended doing this since it way allows us to make versioned changes to the Dockerfile/Makefile/etc without having to update the version of flow